doc/user_guide/interactions/parameters.rst

```diff
@@ -0,0 +1,485 @@
+.. currentmodule:: altair
+
+.. _parameters:
+
+Parameters, Conditions, & Filters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I don't love the length of this heading, but couldn't figure out a way to split it further since we go from params to conditions/filters, and then back to params in this page. I think this is fine but just wanted to point it out.